app/actors/hyrax/actors/file_actor.rb

```diff
@@ -5,62 +5,66 @@ class FileActor
       attr_reader :file_set, :relation, :user
 
       # @param [FileSet] file_set the parent FileSet
-      # @param [String] relation the type/use for the file.
+      # @param [Symbol, #to_sym] relation the type/use for the file
```

Comment:
It's not materially different.  String is fine. Symbol is fine.  The point is that it is only used as a thing that gets `.to_sym` called on it anyway.